ftplugin/agda.vim

```diff
@@ -1,4 +1,9 @@
-call nvimhs#start(expand('<sfile>:p:h:h'), 'cornelis', ['-v', 'DEBUG', '-l', '/tmp/cornelis.log'])
+if exists("g:cornelis_use_global_binary")
+  call remote#host#Register('cornelis', '*', rpcstart('cornelis', []))
+else
```

Comment:
In the future we could allow people to provide the global binary path as a string, but for now this works for me!

Reply:
Might be a good thing to tackle at the same time as https://github.com/isovector/cornelis/issues/12

Reply:
good call